src/main/webapp/src/app/Login-SignUp-page/Login-SignUp.component.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component} from '@angular/core';
+import {SocialAuthService} from 'angularx-social-login';
+import {GoogleLoginProvider} from 'angularx-social-login';
+import {HttpClient} from '@angular/common/http';
+import {HttpHeaders, HttpParams} from '@angular/common/http';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'login',
+  templateUrl: './Login-SignUp.component.html',
+  styleUrls: [
+    './Login-SignUp.component.css',
+    '../common/growpod-page-styles.css',
+  ],
+})
+export class LoginComponent {
+  user: any;
+
+  constructor(
+    private authService: SocialAuthService,
+    private httpClient: HttpClient,
+    private router: Router
+  ) {}
+
+  /**
+   * @param data user data taken from google account: email and name
+   */
+
```

Comment:
nit: remove space between comment block and first line of function. This applies in several locations.

Reply:
done!